clients/core-java/src/main/java/org/commonjava/indy/client/core/IndyClientHttp.java

```diff
@@ -623,7 +623,7 @@ public void close()
     public void delete( final String path )
             throws IndyClientException
     {
-        delete( path, HttpStatus.SC_NO_CONTENT );
+        delete( path, HttpStatus.SC_NO_CONTENT, HttpStatus.SC_OK );
```

Comment:
204 should be an allowable response for DELETE as well. This is fairly common in REST services.

Reply:
Yes. This code allows both 204 and 200.

Reply:
As I remember it, I did this change because some test returns 200 for deletion. So I added 200 to the allowable code list.

Reply:
Ugh, sorry...didn't read closely enough.